include/fmt/core.h

```diff
@@ -755,8 +755,11 @@ class compile_parse_context
   using base::check_arg_id;
 
   FMT_CONSTEXPR void check_dynamic_spec(int arg_id) {
+    detail::ignore_unused(arg_id);
+#if !defined(__LCC__)
```

Comment:
Do we need to exclude `__NVCOMPILER` here too?

Reply:
I don't know.
My tests have shown that this set of ``ifdef``s is sufficient for Nvidia, but I do not use the Nvidia compiler in production and do not know its specifics.

LCC supports ``is_constant_evaluated`` but Nvidia does not. Therefore, for Nvidia, disabling the test is enough.

This ``ifdef`` disables code for LCC, as there is an LCC specific bug (Internal compiler error).

LCC and Elbrus is my hobby, not production usage, but I have access to it for testing.